Sharpmake/Util.cs

```diff
@@ -39,6 +39,14 @@ public static partial class Util
 
         public static readonly char OtherSeparator = UsesUnixSeparator ? Util.WindowsSeparator : Util.UnixSeparator;
 
+        public const char SpaceChar = ' ';
+        public const char SimpleQuoteChar = '\'';
+        public const char DoubleQuotesChar = '\"';
+        public static readonly string Space = @" ";
+        public static readonly string SimpleQuote = @"'";
+        public static readonly string DoubleQuotes = @"""";
+        public static readonly string EscapedDoubleQuotes = @"\""";
```

Comment:
Why did you switch to static readonly strings, not const?

Reply:
I thought only char could be const. String changed to const in 545e4469ee2a7c96818020c2581778babf82d4de